ui/src/commonMain/kotlin/com/materiiapps/gloom/ui/utils/ModifierUtils.kt

```diff
@@ -0,0 +1,6 @@
+package com.materiiapps.gloom.ui.utils
+
+import androidx.compose.ui.Modifier
+
+inline fun Modifier.thenIf(predicate: Boolean, block: Modifier.() -> Modifier)
+    = then(if (predicate) block() else Modifier)
```

Comment:
unnecessary blank modifier
```suggestion
    = if (predicate) then(block()) else this
```